### PR TITLE
API docs for `EnterAcceptsDirective`: don't refer to private fields

### PR DESCRIPTION
**Pull request by chalin**
... and other minor tweaks.

cc @kwalrath 
cf https://github.com/dart-lang/site-webdev/pull/608/commits/9cf652e817b1e6418542f828f9f2bf91c51e281a

### CODE REVIEW

**Comment by chalin** [comment]
@nshahan - Travis reports an error but I don't think that it is related to the changes made here.

File: lib/src/components/material_yes_no_buttons/material_yes_no_buttons.dart [hunk 1]
```diff
@@ -266,8 +266,8 @@ class EscapeCancelsDirective extends BoundaryAwareKeyUpDirective
   void _onMatchingKeyUp(KeyboardEvent event) => _yesNo.no.add(true);
 }
 
-/// If attached to the yes-no buttons it will listen for enter`keyup` event
-/// and trigger [yes] action on it.
+/// If attached to yes-no buttons, it will listen for enter `keyup` events
```
Comment (kwalrath) [inline lib/src/components/material_yes_no_buttons/material_yes_no_buttons.dart]: enter -> Enter? Enter/Return?

(I was trying to parse "enter" as a verb at first.)

Comment (chalin) [inline lib/src/components/material_yes_no_buttons/material_yes_no_buttons.dart]: Changed to: Enter

**Comment by nshahan** [comment]
Thanks, I will work on getting these changes merged internally.

**Comment by nshahan** [comment]
Thanks for the update. These changes landed in 6efe993bb141d69e1f4f4920bb82eae60fb3c041.

**Pull request closed without merging**